### PR TITLE
[patch] fix-pipeline + update dependency

### PR DESCRIPTION
**Pull request by WindzCUHK**
as title

---
[lestrrat-go/jwx](https://github.com/lestrrat-go/jwx/releases/tag/v1.0.0) after 1.0.0
>`jwk.Materialize` has been renamed to `Raw()`
---
close https://github.com/yahoojapan/athenz-authorizer/issues/62

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,6 +28,7 @@ jobs:
       - run:
           name: run tests
           command: |
+            rm -rf example
```
Comment (kyfujisa) [inline .circleci/config.yml:31]: What is this needed for?

Comment (WindzCUHK) [reply .circleci/config.yml:31]: to be consistent with https://github.com/yahoojapan/authorization-proxy/blob/master/.circleci/config.yml#L32

I think previously with we provide example code in `/example`.
Remove in all repo if we don't need it.

Comment (kyfujisa) [reply .circleci/config.yml:31]: ok, understand

File: jwk/daemon.go [hunk 2]
```diff
@@ -134,9 +134,11 @@ func (j *jwkd) getKey(keyID string) interface{} {
 		return nil
 	}
 
-	for _, keys := range j.keys.Load().(*jwk.Set).LookupKeyID(keyID) {
-		raw, err := keys.Materialize()
-		if err == nil {
+	for _, key := range j.keys.Load().(*jwk.Set).LookupKeyID(keyID) {
+		var raw interface{}
+		if err := key.Raw(&raw); err != nil {
```
Comment (kyfujisa) [inline jwk/daemon.go:139]: related release: https://github.com/lestrrat-go/jwx/releases/tag/v1.0.0

> `jwk.Materialize` has been renamed to `Raw()`.

File: .circleci/config.yml (hunk 1 above)
Comment (WindzCUHK) [reply .circleci/config.yml:31]: may be better to remove it?

Comment (kyfujisa) [reply .circleci/config.yml:31]: I think it's fine as it is now. No problem.

**Comment by kyfujisa** [comment]
Other than this, I think it's OK.

**Comment by kyfujisa** [comment]
@wfan
The additional documentation is LGTM.

Resolve CI errors.

Indentation and file name errors, but I think you can disable them.

**Pull request merged**